specification/vmwarevirtustream/resource-manager/Microsoft.VMwareVirtustream/preview/2016-09-01-preview/vmwarevirtustream.json

```diff
@@ -0,0 +1,1438 @@
+          "Clusters"
+        ],
+        "summary": "modify cluster",
+        "consumes": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "name": "clusterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the cluster."
+          },
+          {
+            "name": "parameters",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/ClusterRequest"
+            }
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "successful operation",
+            "schema": {
+              "$ref": "#/definitions/ClusterResponse"
+            }
+          },
+          "202": {
+            "description": "accepted operation"
+          }
+        },
+        "x-ms-examples": {
+          "Clusters_Update": {
+            "$ref": "./examples/Clusters_Update.json"
+          }
+        }
+      },
+      "delete": {
+        "operationId": "Clusters_Delete",
+        "x-ms-long-running-operation": true,
+        "tags": [
+          "Clusters"
+        ],
+        "summary": "delete cluster",
+        "parameters": [
+          {
+            "$ref": "#/parameters/subscriptionId"
+          },
+          {
+            "$ref": "#/parameters/resourceGroupName"
+          },
+          {
+            "name": "privateCloudName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the private cloud."
+          },
+          {
+            "name": "clusterName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "The name of the cluster."
+          },
+          {
+            "$ref": "#/parameters/apiVersion"
+          }
+        ],
+        "responses": {
+          "202": {
+            "description": "accepted operation"
+          },
+          "204": {
+            "description": "no content"
+          }
+        },
+        "x-ms-examples": {
+          "Clusters_Delete": {
+            "$ref": "./examples/Clusters_Delete.json"
+          }
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.VMwareVirtustream/privateClouds/{privateCloudName}/operationresults/{operationId}": {
+      "get": {
+        "operationId": "PrivateClouds_OperationResultsGetById",
```

Comment:
I don't see an `x-ms-long-running-operation` however the 202 response and the description of the `operationId` parameter makes it sound like it is async?

Reply:
So, I think both of these operation APIs should be removed, now that I've added `x-ms-long-running-operation`. I'm going to remove them. We are not using them directly from our clients.